common/src/main/res/values/strings.xml

```diff
@@ -1025,10 +1025,11 @@
     <string name="widget_spinner_server">Server:</string>
     <string name="widget_state_separator_label">State and attribute separator:</string>
     <string name="widget_static_image_description">Entity state</string>
+    <string name="widget_graph_image_description">Statistics widget</string>
     <string name="widget_tap_action_label">Tap action:</string>
     <string name="widget_tap_action_toggle">Toggle</string>
     <string name="widget_template_error">Unable to render the template</string>
-    <string name="widget_text_hint_label">Label</string>
+    <string name="widget_text_hint_label">Entity name</string>
```

Comment:
This change may make sense for widgets but its also used in the Assist shortcut where this won't make sense

https://github.com/home-assistant/android/blob/15b1518a4466e9c58d26efce7e4605858b88a689/app/src/main/java/io/homeassistant/companion/android/widgets/assist/AssistShortcutView.kt#L64